Cargo.toml

```diff
@@ -70,3 +75,19 @@ facebook          = ["hyper/ssl", "openssl"]
 portaudio-backend = ["portaudio"]
 pulseaudio-backend= ["libpulse-sys"]
 default           = ["with-syntex", "portaudio-backend"]
+
+[package.metadata.deb]
+maintainer = "nobody"
+copyright = "2015 Paul Lietar"
+license_file = ["LICENSE", "0"]
```

Comment:
Nevermind, according to the `cargo-deb` README
> license_file: The location of the license and the amount of lines to skip at the top

However, `cargo-deb` specifies `license_file = ["LICENSE", "4"]`, but uses the exact same license file (modulo the author name)